tests/Support/FileHelper.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Queue\AMQP\Tests\Support;
+
+use RuntimeException;
+
+final class FileHelper
+{
+    /**
+     * @param string $fileName File to be created
+     * @param int|string $data Data to be written into the file
+     */
+    public function put(string $fileName, int|string $data): void
+    {
+        if (file_put_contents("{$this->getRuntimeDir()}/$fileName", $data, FILE_APPEND) === false) {
+            throw new RuntimeException("Runtime dir {$this->getRuntimeDir()} or file $fileName are not writable.");
+        }
+    }
+
+    public function get(string $filename): ?string
+    {
+        $path = $this->getFilePath($filename);
+        if (!file_exists($path)) {
+            return null;
+        }
+
+        $result = file_get_contents($path);
+        if ($result === false) {
+            throw new RuntimeException("File $path exists but not readable.");
```

Comment:
```suggestion
            throw new RuntimeException("File $path exists but is not readable.");
```

Reply:
I'd suggest `File "$path" exists but is not readable`